main.go

```diff
@@ -39,7 +40,15 @@ var (
 )
 
 func init() {
+	shortVer := strings.Split(Version, "+")[0]
+	shortVer = strings.Replace(shortVer, ".", "", -1)
```

Comment:
```suggestion
	shortVer = strings.ReplaceAll(shortVer, ".", "")
```

Reply:
And it would cause problem for version `1.1.60` and `1.16.0` .....

Reply:
The reference issues talks a bit about this... It seems like a label is not plausible as a fix(I don't see a reason why tho), but that would be more or less the only way to avoid such problem.

Reply:
you could do: `100²*major+100*minor+patch` - as long as we dont have any number > 99 it would work - if you make add an extra bit of space just use 1000 instead of 100 ...

Reply:
yes, I haven't got the point why the numerical version field is needed. If the real requirment is to see about `new version update`, a field like `newVersionAvailable` is enough. If we use some trick to output numerical version, how could it be used ..... is it really useful